token/client/src/token.rs

```diff
@@ -281,17 +281,15 @@ where
         ))))
     }
 
-    fn get_multisig_signers<S: Signers>(
+    fn get_multisig_signers<'a>(
         &self,
-        authority: &Pubkey,
-        signing_keypairs: &S,
-    ) -> Vec<Pubkey> {
-        let signing_pubkeys = signing_keypairs.pubkeys();
-
-        if signing_pubkeys == [*authority] {
+        authority: &'a Pubkey,
+        signing_pubkeys: &'a Vec<Pubkey>,
+    ) -> Vec<&'a Pubkey> {
```

Comment:
nit only because you mentioned this was your first time annotating lifetimes:

since the return is a reference to `signing_pubkeys`, you can do a slightly more flexible version
```
fn get_multisig_signers<'a, 'b>(
        &self,
        authority: &'b Pubkey,
        signing_pubkeys: &'a Vec<Pubkey>,
    ) -> Vec<&'a Pubkey> {
```

This means that the lifetimes for `authority` and `signing_pubkeys` don't need to be the same. Most likely this would almost never be useful since it's called internally, but thought you like the info :smile: